src/components/tooltip/Tooltip.scss

```diff
@@ -56,13 +56,13 @@ $tooltip-offset: 4px; /* amount of space between target and tooltip arrow */
     }
 
     &.with-close-button {
-        padding-right: 24px;
+        padding-right: 28px;
     }
 
     .tooltip-close-button {
         position: absolute;
-        top: 4px;
-        right: 0;
+        top: 10px;
```

Comment:
this doesn't look centered in the screenshot - would it make sense to use flex here to get it exactly right and handle multiple lines better?

Reply:
I'm not sure if flexbox will help here. The close button needs to be centered with the first line of text, whether it's the only line or one of many in the tooltip.

The other issue is that depending on how the text is capitalized it looks less or more centered. I added more realistic text (text we actually show) to test the alignment, and I moved the close button up one pixel. Looks like this now:

<img width="491" alt="Screen Shot 2019-10-25 at 9 44 19 AM" src="https://user-images.githubusercontent.com/1280912/67589273-e549d480-f70c-11e9-8748-2ca6879ea21b.png">